### PR TITLE
increased the number of tries in kube-proxy test

### PR DESCRIPTION
**Pull request by ArtfulCoder**
After the iptables-proxy, we need a few more tries to verify round-robin amongst endpoints.

### CODE REVIEW

**Comment by ArtfulCoder** [comment]
should help with #17781

File: test/e2e/kubeproxy.go [hunk 1]
```diff
@@ -150,7 +150,7 @@ func createHTTPClient(transport *http.Transport) *http.Client {
 
 func (config *KubeProxyTestConfig) hitClusterIP(epCount int) {
 	clusterIP := config.nodePortService.Spec.ClusterIP
-	tries := epCount*epCount + 5 // if epCount == 0
+	tries := epCount*epCount + 15 // if epCount == 0
```
Comment (mikedanese) [inline test/e2e/kubeproxy.go]: what about line 172

Comment (ArtfulCoder) [inline test/e2e/kubeproxy.go]: done

**Comment by wojtek-t** [comment]
LGTM - thanks for doing this

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by mikedanese** [comment]
@ArtfulCoder please rebase

**Comment by ArtfulCoder** [comment]
rebase done

**Comment by wojtek-t** [comment]
@ArtfulCoder - it seem that you didn't push it

**Comment by ArtfulCoder** [comment]
its pushed now.

**Pull request merged**